plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/PlayEffectCommand.java

```diff
@@ -86,6 +87,10 @@ public PlayEffectCommand() {
     // - For BLOCK_MARKER, FALLING_DUST, BLOCK_CRACK, or BLOCK_DUST particles, the input is any valid MaterialTag, eg "stone".
     // - For VIBRATION, the input is <duration>|<origin>|<destination> where origin is a LocationTag and destination is either LocationTag or EntityTag, for example "5s|<context.location>|<player>"
     // - For ITEM_CRACK particles, the input is any valid ItemTag, eg "stick".
+    // - For TARGET_COLOR, the input is of format: <color>|<location>, for example "red|<player.cursor_on>". Color input is any valid ColorTag object.
+    // - For COLOR, the input is any valid ColorTag.
+    // - For FLOAT, the input is any valid ElementTag(Decimal).
+    // - For INTEGER, the input is any valid ElementTag(Integer).

```

Comment:
it'd be `(Number)` but also just say `any valid number`, don't be overly technical when you don't need to be

Also what the heck is an integer particle? That `For x y z particles,` half should name particles that use a format, not the format's java class name